Nataly_Bondarenko/task2_arif.op.py

```diff
@@ -0,0 +1,11 @@
+num = 2
+
+print(num * 2)
+
+num *= 2
+print(num)
+
+print(num / 2)
+
+num /= 2
+print(num)
```

Comment:
use byte operator to divide and multiply by 2

Reply:
changed it